main.go

```diff
@@ -0,0 +1,28 @@
+package main
+
+import (
+	"fmt"
+	"log"
+	"net/http"
+
+	"github.com/RealImage/QLedger/internal/utils/config"
```

Comment:
Don't bother pulling stuff out to sub packages until necessary. For now just a main.go file is more than enough. `httprouter` is good.